Userland/Libraries/LibCore/Stream.h

```diff
@@ -434,8 +433,8 @@ class TCPSocket final : public Socket {
 
 class UDPSocket final : public Socket {
 public:
-    static ErrorOr<NonnullOwnPtr<UDPSocket>> connect(DeprecatedString const& host, u16 port, Optional<Time> timeout = {});
-    static ErrorOr<NonnullOwnPtr<UDPSocket>> connect(SocketAddress const& address, Optional<Time> timeout = {});
+    static ErrorOr<NonnullOwnPtr<UDPSocket>> connect(DeprecatedString const& host, u16 port, Optional<Time> timeout);
+    static ErrorOr<NonnullOwnPtr<UDPSocket>> connect(SocketAddress const& address, Optional<Time> timeout);
```

Comment:
Why are we making the timeout required?

Reply:
Because out of the total of two callsites to this static method, one already provides the argument anyway. That doesn't justify the cost of basically all of Serenity having to include Time.h.

Removing the include of Time.h from Stream.h is the entire point of this PR, because it reduces (re)build times.